coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 81.54
+    "covered_percent": 77.12
```

Comment:
I'm using mocks instead of calling methods from gtfs_parser that aren't related to multiple stops.